### DISCUSSION TITLE
webpack dev server throws error "[webpack-cli] Error: Unknown option '--open'"

### DISCUSSION THREAD

**Discussion by arunvenkat1911** [webpack/webpack#17364] [Q&A]
I am using these versions
Node 16.3.0
npm 7.15.1

When I use this command 
"set NODE_OPTIONS=--max_old_space_size=8192 && webpack serve --open --config ./webpack.config.js --progress"

I am getting this error "[webpack-cli] Error: Unknown option '--open'"

Please assist

**Comment by alexander-akait** [comment]
Please run `npm ls webpack-cli`
